packages/livekit-rtc/src/nodejs.rs

```diff
@@ -13,10 +12,16 @@ use napi::{
 };
 use napi_derive::napi;
 use prost::Message;
-use std::sync::Arc;
+use std::sync::{Arc, OnceLock};
+
+static SDK_VERSION: OnceLock<String> = OnceLock::new();
```

Comment:
is there a reason to store it statically?

Reply:
No it seems to work just as well with const. I don't think it makes any difference but I updated it anyways.

Reply:
actually I can't figure out why I thought I needed to store this here anyways. it's being stored in the server's FfiConfig and that's all it needs.